sql/dql/intro-joins/relative-database-intro.md

```diff
@@ -0,0 +1,108 @@
+---
+author: meridian
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - joins
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+---
+
+# Relational Databases
+
+---
+
+## Content
+
+You are now familiar with querying individual tables.
+
+Real databases are often comprised of more than one table.
+
+Relational databases consist of tables and entries that connected to one another in some way. We call this connection a *relationship*.
+
+Consider the `pokedex` database, it contains two tables:
+
+`pokemon_health`
+| id | name       | health |
+|----|------------|--------|
+| 1  | Venusaur   | 42     |
+| 2  | Charmander | 35     |
+
+`pokemon_attack`
+| id | type   | base-attack |
+|----|--------|-------------|
+| 1  | poison | 21          |
+| 2  | fire   | 15          |
+
+Inspecting them, we can easily see that the `id` column is shared.
+
+More importantly, these two tables **map**, or are related, to one another.
+
+So the `attack` and type data for Venusaur (`id` = 1) is contained in the `pokemon_attack` table, while data for the `name` and the `health` is contained in the `pokemon_health` table[1].
+
+![Health and Attack Tables Graphic](https://img.enkipro.com/2d381a48dbcfcb92827e8fca3b95c450.png)
+
+The **relation** between these two tables is the `id` column and can be used to join them.
+
+> 💡 Note that these two tables do **not** represent best-practices for database design, the reasons for this are covered in the [**Defining Data**](https://app.enkipro.com/course/ddl) topic.
```

Comment:
let's use app.enki.com because app.enkipro.com is the legacy url
```suggestion
> 💡 Note that these two tables do **not** represent best-practices for database design, the reasons for this are covered in the [**Defining Data**](https://app.enki.com/course/ddl) course.
```